Fachportal Padagogik.js

```diff
@@ -326,6 +378,195 @@ var testCases = [
+				"shortTitle": "Hooper, S.R./ Willis, G. (1989)",
+				"volume": "38",
+				"attachments": [
+					{
+						"title": "Full Text PDF"
+					},
+					{
+						"title": "Snapshot"
+					},
+					{
+						"title": "Link",
+						"snapshot": false
+					}
+				],
+				"tags": [
+					"Book review",
+					"Classification",
+					"Classification system",
+					"Klassifikationssystem",
+					"Learning Difficulties",
+					"Learning Difficulty",
+					"Learning disorder",
+					"Lerndefizit",
+					"Lernschwierigkeit",
+					"Lernschwäche",
+					"Review",
+					"Rezension"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.pedocs.de/frontdoor.php?source_opus=11837",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Die diskursanalytische Rekonstruktion von politischen Leitbildern bildungsbezogener 'guter Kindheit'",
+				"creators": [
+					{
+						"firstName": "Stefanie",
+						"lastName": "Bischoff",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Tanja",
+						"lastName": "Betz",
+						"creatorType": "author"
+					}
+				],
+				"date": "2011",
+				"abstractNote": "Ziel der diskursanalytischen Betrachtung von politischen Dokumenten (hierzu gehören: Pläne, Programme und Berichte in bildungs&#8208; und kindheitsrelevanten Politikfeldern) und des darin auftretenden Diskurses zum Thema gute Kindheit und Bildung ist es, sie daraufhin zu untersuchen, welche Vorstellungen/Wissensbestände von guter Kindheit (und damit verbunden richtiger  Erziehung und Bildung, guter Elternschaft, guter pädagogischer Arbeit in Bildungsinstitutionen) transportiert werden. Das Projekt beabsichtigt daher, die politisch hervorgebrachten Leitbilder einer frühen und öffentlich verantworteten Bildungskindheit (vgl. Betz 2010) und ihre Konnotationen zu rekonstruieren. Ein Ausgangspunkt ist, dass die in den Dokumenten erzeugten Annahmen, Strategien und Maßnahmen einen weitreichenden Einfluss auf bildungspolitische Veränderungen und die Ausgestaltung der öffentlich verantworteten (Früh&#8208;) Erziehung/Bildung haben, also soziale Prozesse aktiv mitgestalten, wie dies beispielhaft an der Implementierung der Bildungs&#8208; und Erziehungspläne oder der Initiierung von Elternbildungsprogrammen veranschaulicht werden kann (siehe auch Kapitel 4.3.2)). Weiter wird davon ausgegangen, dass diese wirkmächtigen Annahmen auch Einfluss auf die Strukturierung der Vorstellungen, Haltungen und Praktiken der sozialen Akteure haben, was es empirisch einzufangen gilt. Der Frage, ob und in welcher Art und Weise dies der Fall ist, inwieweit hier die sozial situierten im Sinne von milieuspezifisch unterschiedlich geprägten Vorstellungen guter Kindheit mit den Vorstellungen in politischen Berichten korrespondieren und welche Bedeutung dies wiederum für die Reproduktion von Bildungsungleichheiten hat, soll [hier] explorativ nachgegangen werden. (DIPF/Orig.)",
```

Comment:
Here you see some problems with the encoding:
`&#8208;` instead of `-`   (derived from `%26%238208%3B`)
`` instead of `‚`  (derived from `%C2%82`)
``instead of `‘`   (derived from `%C2%91`)

I will try to replace these cases before doing more with the `coins`.